pkgs/servers/x11/xorg/generate-expr-from-tarballs.pl

```diff
@@ -73,8 +74,13 @@
         next;
     }
 
+    # split by first occurence of only numbers and dots surrounded by hyphens or end of line
+    my ($name, $version) = split(/(-[\.0-9]+(?:$|-(?:.+)))/, $pkgName, 2);
```

Comment:
- There's an unlikely corner case if the closing hyphen is the last character. Itches me a little bit so...
```suggestion
    my ($name, $version) = split(/(-[\.0-9]+(?:$|-(?:.*)))/, $pkgName, 2);
```

Reply:
i did that on purpose. thought that this prevents some PRs, because it will look bad. But I am fine in changing this.